agent/api/container.go

```diff
@@ -78,3 +194,12 @@ func (c *Container) SetSentStatus(status ContainerStatus) {
 
 	c.SentStatus = status
 }
+
+// String returns a human readable string representation of this object
+func (c *Container) String() string {
+	ret := fmt.Sprintf("%s(%s) (%s->%s)", c.Name, c.Image, c.GetKnownStatus().String(), c.GetDesiredStatus().String())
```

Comment:
I'm not a big fan of how we log this information, especially because of the very long log lines that show up because of this. I think we can use the "task-id" as a marker everywhere we print this (and the task.String()) and only emit information about name, image state when asked for/needed. I'll create an issue to revisit this in the future.